spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -605,31 +605,41 @@ private static String sqlString(org.apache.iceberg.expressions.Literal<?> lit) {
   }
 
   public static CatalogAndIdentifier catalogAndIdentifier(SparkSession spark, String name) throws ParseException {
+    return catalogAndIdentifier(spark, name, spark.sessionState().catalogManager().currentCatalog());
+  }
+
+  public static CatalogAndIdentifier catalogAndIdentifier(SparkSession spark, String name,
+                                                            CatalogPlugin fallBackCatalog) throws ParseException {
```

Comment:
nit: formatting is off

Reply:
I wish I could teach IDEA this, every time I run reformat it swaps it back